CHANGELOG.md

```diff
@@ -4,6 +4,9 @@
 
   - Dropped support for Ruby 3.0
 
+  - Update `Logster::Logger#chain` to accept a `with_opts` keyword argument. When set to `true`, an instance of `Logster::Logger`
```

Comment:
It's a bit unfortunate we called them `opts` and therefore `with_opts`. `with_opts` makes me think we're chaining with options instead of `include_diagnostics` which is a keyword I would prefer.